package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@eik/postcss-import-map",
+  "version": "1.0.0",
+  "description": "PostCSS plugin that uses Eik defined import map files to transform bare import specifiers to absolute URLs in @import rules",
+  "main": "dist/plugin.cjs.js",
+  "files": [
+    "src",
+    "dist"
+  ],
```

Comment:
Add README, LISENCE and CHANGELOG.

Reply:
And package.json

Reply:
README, LISENCE and package.json are always included by npm. Verified by running npm pack locally. I'll add CHANGELOG 🙂